---
rws_pr2_navigation/Location[] locations